ewatercycle/parameter_sets/default.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+from typing import Set, Optional
+
+from ewatercycle import CFG
+
+
+class ParameterSet:
+    """Container object for parameter set options.
+
+    Attributes:
+        name (str): Name of parameter set
+        directory (Path): Location on disk where files of parameter set are stored.
+            If Path is relative then relative to CFG['parameterset_dir'].
+        config (Path): Model configuration file which uses files from :py:attr:`~directory`.
+            If Path is relative then relative to CFG['parameterset_dir'].
+        doi (str): Persistent identifier of parameter set. For a example a DOI for a Zenodo record.
+        target_model (str): Name of model that parameter set can work with
+        supported_model_versions (Set[str]): Set of model versions that are supported by this parameter set.
+            If not set then parameter set will be supported by all versions of model
+    """
+
+    def __init__(
+        self,
+        name: str,
```

Comment:
In the example API notebook, `id` is used instead of `name`. Maybe change it, or support both?

```py
@property
def id(self):
    return self.name
```

Reply:
`id` is a built-in. In PyCharm I get `Shadows built-in name 'id' ` warning. I think we should stick with `name`.